subsys/bootloader/image/CMakeLists.txt

```diff
@@ -70,14 +70,21 @@ if (CONFIG_SB_VALIDATE_FW_SIGNATURE AND
   set(PROVISION_DEPENDS signature_public_key_file_target)
 endif()
 
+if (CONFIG_SOC_NRF5340_CPUNET)
```

Comment:
We should consider moving these out to a separate commit, but IMO it can stay here.